sourcetracker/sourcetracker.py

```diff
@@ -843,3 +855,106 @@ def _cli_loo_runner(sample, source_category, alpha1, alpha2, beta, restarts,
     o = open(os.path.join(output_dir, sample + '.txt'), 'w')
     o.writelines(lines)
     o.close()
+
+
+def _api_gibbs(source_df, sink_df, alpha1, alpha2, beta, restarts,
```

Comment:
i made it `_gibbs` so that it maintains the 'private method' convention of a leading underscore. When we refactor we can make the API call what the actual script depends on, and thus eliminate this.